index.js

```diff
@@ -1,34 +1,36 @@
+'use strict';
 var Readable = require('stream').Readable;
-var _ = require('underscore');
 var request = require('request');
 
-var pagerduty = function(subdomain, token) {
+var pagerduty = function (subdomain, token, url) {
     this.subdomain = subdomain;
     this.token = token;
+    this.url = url;
     return this;
 };
 
 module.exports = pagerduty;
 
-pagerduty.prototype.stream = function(status, services, interval) {
+pagerduty.prototype.stream = function (options, interval) {
     var that = this;
     var stream = new Readable({objectMode: true});
     var seen = {};
     var calling = false;
 
-    stream._read = function() {
+    stream._read = function () {
         if (calling) return;
         query();
     };
 
-    var query = function() {
+    var query = function () {
         calling = true;
-        that.getIncidents(status, services, function(err, incidents) {
+        that.getIncidents(options, function (err, data) {
+            var incidents = data.incidents;
             if (err) return stream.emit('error', err);
             for (var i = 0; i < incidents.length; i++) {
                 var incident = incidents[i];
                 if (!seen[incident.id]) {
-                    stream.push(incident);
+                    stream.push(JSON.stringify(incident, null, 2));
```

Comment:
Is there a reason for stringifying here?  I'm just curious whether it is for convenience, or another reason.